WordPress/src/main/res/drawable/bg_rectangle_warning_surface_with_padding.xml

```diff
@@ -2,21 +2,21 @@
 <layer-list xmlns:android="http://schemas.android.com/apk/res/android">
     <item>
         <shape android:shape="rectangle">
-            <solid android:color="@android:color/white" />
+            <solid android:color="?attr/colorSurface" />
         </shape>
     </item>
     <item
-        android:left="@dimen/margin_extra_large">
+        android:left="@dimen/notification_comment_padded_frame_bg_offset">
         <shape android:shape="rectangle">
-            <solid android:color="@color/warning" />
+            <solid android:color="?attr/wpColorWarningLight" />
             <padding
                 android:bottom="1dp"
```

Comment:
Is there a reason these values were hardcoded? I know it's outside the scope of this PR to modify this so they could be left as is. Let me know.

Reply:
Those values are a "hackish" way to play with layers to get the required drawable. I don't think they should be exposed in dimens, since describing them in a variable name without confusing someone non-familiar with them is hard. I exposed `notification_comment_padded_frame_bg_offset` and `notification_comment_frame_bg_border_width` that you can play with to change the drawable border.

Reply:
Okay understood! Makes sense 🙏